src/apply-types.ts

```diff
@@ -81,29 +67,20 @@ export function applyTypesToFile(
             replacements.push(Replacement.insert(opts.parens[0], '('));
             suffix = ')';
         }
+        if (opts && opts.comma) {
+            suffix = ', ';
```

Comment:
`opts.parens` is only used for one parameter arrow functions so this should never happen since arrow functions use the surrounding `this` and therefore don't need an explicit `this` type.